ui/imports/shared/controls/chat/UserImage.qml

```diff
@@ -16,9 +17,12 @@ Loader {
     property string name
     property string pubkey
     property string image
-    property bool showRing: false
+    property bool showRing: true
     property bool interactive: true
 
+    property int colorId: Utils.colorIdForPubkey(pubkey)
+    property var colorHash: Utils.getColorHashAsJson(pubkey)
```

Comment:
I see you overwrite those properties all the time, so are the default values needed? I feel like the Utils function are not really that useful since we already use a model.
My concern is that someone might use the "easy" way of using the Utils function in the future instead of using a model.

Reply:
These particular properties were private before, but I needed to expose them because of https://github.com/status-im/status-desktop/pull/5424#discussion_r845840817. Because on the login page there is no access to pubkeys I need to override these with model data. This is the only place where `UserImage::colorId` and `UserImage::colorHash` are overridden.